src/Polysemy/Internal/Union.hs

```diff
@@ -111,22 +110,21 @@ weave
     -> (∀ x. s x -> Maybe x)
```

Comment:
While you're at it, can you remove the redundant `Functor m`  constraint here?

Reply:
Done